### ISSUE TITLE
[Shows Card] - Play Button Accessibility Issues

### ISSUE THREAD

**Issue by garyb1** [syntaxfm/website#1449]
The Play button for each card on the [shows page](https://syntax.fm/shows) is very difficult to see. It has a really low contrast ratio of about 1.29:1. It would actually need at least 3:1 ratio or more.

<img width="1205" alt="Play button for syntax podcasts" src="https://github.com/syntaxfm/website/assets/12486130/b47cd614-5f5c-42a2-9405-9d01615ce776">

<img width="495" alt="Play button with focus-visible outline" src="https://github.com/syntaxfm/website/assets/12486130/c6be2099-7a37-4954-90db-3e7839550ae9">

The button also has no accessible name. We just have a title element with the correct name, but it is a child of an element containing aria-hidden="true".

<img width="504" alt="code screenshot showing button with no accessible name" src="https://github.com/syntaxfm/website/assets/12486130/9b5111e5-8761-452a-878d-2f351fcfa671">

Happy to help with fixes.

**Comment by themisterholliday** [comment]
Seems like this could be lumped in with https://github.com/syntaxfm/website/issues/1359

**Comment by stolinski** [comment]
Yeah this seems like a straight up bug. Should be white.
